lib/migration-stream.js

```diff
@@ -72,6 +79,17 @@ function MigrationStream() {
       migrationStream.push(tile);
       return callback();
     }
+    else{
+      if(!v1TileDataLogged && process.env.LOG_V1_TILES){
+        v1TileData['tile_version'] = 'V1';
+        
+        if (v1TileData.tile_version) {
+            console.log('[Debug] - begin writing v1-stats file');
+            fs.writeFileSync('v1-stats.json', JSON.stringify(v1TileData));
```

Comment:
Looking good here @millzpaugh! I think you'll need a `return callback();` once you're done writing, just like the `if (v2 === true)` part above.

Reply:
fixed @mapsam!